language-support/qute/qute.tmLanguage.json

```diff
@@ -362,6 +362,10 @@
         {
           "match": "\\b(const|goto)\\b",
           "name": "keyword.reserved.java"
+        },
+        {
+          "match": "\\b(include)\\b",
+          "name": "keyword.control.import"
```

Comment:
Currently there is no colour difference for every keyword/no notable different shade of pink to use.

Reply:
I think it's not teh correct fix, because if you write `{include}` it should be not a keyword.

Reply:
If this fix is incorrect, then we shouldn't have https://github.com/redhat-developer/vscode-quarkus/blob/master/language-support/qute/qute.tmLanguage.json#L325 either, since syntax like for, if, etc.. is doesn't need to follow a `#` or `/` and is still a keyword.